<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameLauncher</class>
 <widget class="QWidget" name="GameLauncher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>游戏启动器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="pb_startgame">
     <property name="text">
      <string>开始游戏</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="cb_curversion"/>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="le_playername"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
